sql/sys_vars.cc

```diff
@@ -1607,6 +1607,12 @@ static Sys_var_bool Sys_binlog_rows_query(
     DEFAULT(false), NO_MUTEX_GUARD, NOT_IN_BINLOG,
     ON_CHECK(check_session_admin));
 
+static Sys_var_bool Sys_binlog_ddl_query(
+    "binlog_ddl_query_log_events",
+    "Write drop table statements having single table into binary log for",
+    SESSION_VAR(binlog_ddl_query_log_events), CMD_LINE(OPT_ARG), DEFAULT(false),
```

Comment:
Also mention here this optino disableds multi-table table drop, ie DROP TABLE t1,t2.